### PR TITLE
[DOCFIX] Plan and add todos to a new outline for the vagrant docs

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@aaudiber

File: docs/en/deploy/Running-Alluxio-Using-Vagrant.md [hunk 1]
```diff
@@ -5,6 +5,25 @@ nickname: Alluxio on Vagrant
...
+    - Access
+    - Destroy
+    - Advanced Tips
+      -Spot Instance
+  - Deploy using Vagrant on GCE
+    - Launch
+    - Access
+    - Destroy
+
+TODO(david): combine content from Deploy on EC2, GCE, Virtual Box using Vagrant
```
Comment (gpang) [inline docs/en/deploy/Running-Alluxio-Using-Vagrant.md]: Does this mean we will get rid of the other pages?

Comment (yuzhu) [reply docs/en/deploy/Running-Alluxio-Using-Vagrant.md]: yes.

**Comment by yuzhu** [comment]
@gpang @aaudiber PTAL, thanks

**Pull request merged**